prac_06/car.py

```diff
@@ -0,0 +1,39 @@
+"""CP1404/CP5632 Practical - Car class."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=42, name=""):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.fuel = fuel
+        self._odometer = 277
```

Comment:
I think the fuel and _odometer values were just an example to show the required format for the __str__ method, you weren't supposed to actually use them

Reply:
Ah Thank you for clarifying that.